app/templates/orders/new.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="ui two column container stackable doubling left grid" >
```

Comment:
@harshitagupta30 you have used a two column container grid, then made use of only one column. 
And in the subsequent code through out this file, you have repeatedly used `container stackable doubling left grid`
Instead of declaring it over and and over again, please declare the container grid once, and make use of `row` to separate the  content. This will lead to less redundant code.